models/cart-discount/src/cart-discount/cart-discount-draft/presets/sample-data-fashion/percent-off-womens-wear.ts

```diff
@@ -9,8 +10,11 @@ const percentOffWomensWear = (): TCartDiscountDraftBuilder =>
     .empty()
     .value(CartDiscountValueRelativeDraft.random().permyriad(1500))
     .cartPredicate('1 = 1')
-    // TODO: create `target` model
-    .target(undefined)
+    .target(
+      CartDiscountLineItemsTargetDraft.random().predicate(
+        `categories.key contains "bottoms-women" and price.discount.id is not defined`
```

Comment:
Is it worth merging the category preset PR in then updating the key here with the preset?